app/assets/javascripts/error_handler.coffee

```diff
@@ -0,0 +1,23 @@
+email = 'support@roqua.nl'
+message = "Onbekende fout. Mail naar <a href='#{email}'>#{email}</a> voor hulp"
```

Comment:
`mailto:#{email}`

Loggen naar appsignal en zeggen  `we've been notified` is beter,